package/src/languageService/kustoLanguageService.ts

```diff
@@ -1068,6 +1067,13 @@ class KustoLanguageService implements LanguageService {
             KustoLanguageService.toBridgeList([...scalarSymbols, ...tabularSymbols])
         );
         this._script = this._script?.WithGlobals(this._kustoJsSchemaV2);
+
+        // Set parameters is only working with the below code. It didn't used to need this, why does it now?!?
+        // Copy+pasted from setSchema
+        const kustoJsSchema = KustoLanguageService.convertToKustoJsSchema(this._schema);
+        this._kustoJsSchema = kustoJsSchema;
+        this.createRulesProvider(kustoJsSchema, this._schema.clusterType);
```

Comment:
Primary change is here. Other changes are to eliminate async pauses in the `setSchemaFromSchowSchema` method.

Reply:
All three lines are required. If I comment out `this.createRulesProvider` it stops working.